src/hooks/init/check-update.ts

```diff
@@ -44,6 +44,7 @@ const hook: Hook<'init'> = async function ({config}) {
   }
 
   const refreshNeeded = async () => {
+    if (process.env.FORCE_VERSION_CACHE_UPDATE) return true
```

Comment:
Can you make this a scoped env var, by doing this?

```
if (this.config.scopedEnvVarTrue('FORCE_VERSION_CACHE_UPDATE')) return true
```

`this.config` has a few `scopedEnvVar` methods that append the cli's name to the env var. So for `heroku` it would be `HEROKU_FORCE_VERSION_CACHE_UPDATE` and for `sf` it would be `SF_FORCE_VERSION_CACHE_UPDATE`

Reply:
Sure thing! Thanks.